src/components/atoms/Button/styles.css

```diff
@@ -0,0 +1,25 @@
+.button {
+    background-color: #0157ff;
+    width: 174px;
+    height: 30px;
+    display: flex;
+    justify-content: center;
+    border-radius: 4px;
+    border: none;
+}
+
+.buttonIcon {
+    width: 16px;
+    height:100%;
+    vertical-align: middle;
+}
+
+.buttonText {
+    color: white;
+    font-size: 14px;
+    height: 100%;
+    vertical-align: middle; 
+    font-family: 'Source Sans Pro', sans-serif;
```

Comment:
This project doesn't have a variable file to store those static values like colors and font names?

Reply:
I opened a PR adjusting this issue. We need to wait for the PR approve then we could do a rebase